app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -1,6 +1,7 @@
 module API
   module V2
     class AccessRequestsController < API::V2::ApplicationController
+      deserializable_resource :access_request, only: %i[create]
```

Comment:
```suggestion
      deserializable_resource :access_request, only: :create
```

Reply:
Ehhh if we add another one then Rubocop will be like "use `%i[`, it's fine.

Reply:
Yep. Ship it